scripts/webidl/convert.js

```diff
@@ -46,4 +46,4 @@ function onlyIDL(filePath) {
   return path.extname(filePath) === ".idl";
 }
 
-doConversion("lib/jsdom/living/generated/events").done();
+doConversion("lib/jsdom/living/events").done();
```

Comment:
Ah OK, I see. Well eventually I want to avoid us having to update this list explicitly but for now this is fine.